src/style/global.sass

```diff
@@ -31,3 +32,50 @@ ul, ol
     padding-inline: 2rem
     width: 100%
     box-sizing: border-box
+
+.centered-container 
+    display:flex
+    width: 100%
+    justify-content:center
+    align-items:center
+    
+/* classes for buttons (jonathan) */
+.button 
+  padding-top: 0.85em
+  padding-bottom: 0.85em
+  padding-right: 2em
+  padding-left: 2em
+  font-weight: bold
+  border: 1px solid transparent
+  border-radius: 1rem
+  cursor: pointer
+
+.primary-button 
+    background-color: $color-primary
+
+.primary-button:hover
```

Comment:
You can nest inside the `primary-button` class the `:hover` by `&:hover`
[See the documentation](https://sass-lang.com/documentation/style-rules/parent-selector/)